htdocs/core/lib/functions.lib.php

```diff
@@ -8633,6 +8633,9 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 				}
 				if (is_object($object) && $object->element == 'facture') {
 					$substitutionarray['__URL_INVOICE__'] = DOL_MAIN_URL_ROOT."/compta/facture/card.php?id=".$object->id;
+					$substitutionarray['__FACDATE__'] = isset($object->date) ? dol_print_date($object->date, 'daytext', 0, $outputlangs) : '';
+					$substitutionarray['__FACDATELIMREG__'] = isset($object->date_lim_reglement) ? dol_print_date($object->date_lim_reglement, 'daytext', 0, $outputlangs) : '';
```

Comment:
Same, we already have __DATE_DUE__ 
https://wiki.dolibarr.org/index.php?title=Variable_substitution_system#Objects_with_amounts

Reply:
@eldy 

you are almost right, the date format is not the same: day versus daytext.

Reply:
why not __DATE_DUE_DAYTEXT__ and __DATE_DAYTEXT__ ?

Reply:
Why not ?
It's ok for me if we just have to change the variables names